Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version:5.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "SwiftSignalRClient",
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "SwiftSignalRClient",
+            targets: ["SwiftSignalRClient"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/richardpiazza/CZlib", from: "0.5.0")
```

Comment:
Why can't we use libz as in the CocoaPods version? My understanding is that libz is already installed so not sure why it has to be brought in as a dependency.

Reply:
I'll dig into this one a bit. This is here for anyone using the library using SPM. Since the dependency is declared for Cocoapods, I was mirroring that here. I'll double-check that this is still required or not.

Reply:
Found that this dependency wasn't needed after all. (The 'import zlib') in WebSocket.swift though is required. When that import is not there, the library will fail to compile do to 'undefined symbols' that are referenced in the file.

Reply:
:+1: